dev/source-release.sh

```diff
@@ -113,8 +113,8 @@ git archive $release_hash --worktree-attributes --prefix $tag/ -o $projectdir/$t
 
 echo "Signing the tarball..."
 [[ -z "$keyid" ]] && keyopt="-u $keyid"
-gpg --detach-sig $keyopt --armor --output ${projectdir}/${tarball}.asc ${projectdir}/$tarball
-shasum -a 512 ${projectdir}/$tarball > ${projectdir}/${tarball}.sha512
+gpg $keyopt --armor --output ${projectdir}/${tarball}.asc --detach-sig ${projectdir}/$tarball
+shasum -a 512 $tarball > ${projectdir}/${tarball}.sha512
```

Comment:
Not sure if there is a better way to fix this, currently the script kind-of assumes you can run it not in the project root directory. But this would break that assumption. But I cannot find a way to avoid doing this. Maybe I just overlooked some flag?

Another way is to create the file and then remove the `projectdir` part of the path. But that seems to be over-complicating things a lot.

Reply:
I think it's fine to run in the project root directory.